ocsf-import/package.yaml

```diff
@@ -0,0 +1,591 @@
+id: ocsf
+name: OCSF
+author: Tenzir
+author_icon: https://github.com/tenzir.png
```

Comment:
```suggestion
author_icon: https://github.com/tenzir.png
package_icon: https://github.com/ocsf.png
```

Reply:
I added the package icon in https://github.com/tenzir/library/pull/10/commits/44ac94e11c937189cfa89c6df07ad8989e86fd17